src/core/Table/Table.tsx

```diff
@@ -47,6 +47,7 @@ import {
   onTableResizeEnd,
   onTableResizeStart,
 } from './actionHandlers/resizeHandler';
+import VirtualScroll from '../utils/components/VirtualScroll';
```

Comment:
Could import this from utils barrel in line 25

Reply:
You know I dont like barrel import.
What is an advantage of barrel import here apart from less chars?